src/exercism/Grains.java

```diff
@@ -0,0 +1,27 @@
+import java.math.BigInteger;
+
+class Grains {
+
+    BigInteger grainsOnSquare(final int square) {
+        if (square < 1 || square > 64) {
+            throw new IllegalArgumentException("square must be between 1 and 64");
+        }
+        BigInteger base = new BigInteger("2");
+        BigInteger exponent = new BigInteger(String.valueOf(square - 1));
+        return base.pow(exponent.intValue());
```

Comment:
This can be just:
return base.pow(square)

Reply:
It makes no sense what you did above. You converted (square variable) int -> string -> BigInteger -> int